client/app/scripts/superdesk-users/users.js

```diff
@@ -1034,6 +1049,19 @@
                         scope.datelinePreview = scope.preferences['dateline:located'].located;
                     });
 
+                    // A list of category codes that are considered preferred
+                    // by default, unless of course the user changes this
+                    // preference setting.
+                    scope.defaultCategories = Object.freeze({
```

Comment:
Can we drive this through `vocabularies` collection? If this list changes then we need to change the code and deploy.